swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=None):
+        if condition == None:
+            super().__init__(category="Clothing")
+        else:
+            super().__init__( condition=condition, category='Clothing')
+        
+    
+    def __str__(self):
+        return "The finest clothing you could wear."
+    
+    def condition_description(self):
+        return super().condition_description()
```

Comment:
Remember that Python will automatically look up through the parent classes of an instance to locate methods that aren't defined on a child class. So if all we are doing in an implementation is calling the parent implementation, do we need the child implementation at all?